burger/style/main-theme.css

```diff
@@ -0,0 +1,7 @@
+.main-theme {
+    --font-color-orange: #F2AA3D;
+    --font-color-orange2: #d1830f;
+    --color-word: white;
+    --br-color:  black;
+    --br-nav-cont: #333232;
```

Comment:
where is `--color-submit` in this theme?

Reply:
fixed all changes